File: gpt/nivel intermedio/go/edad.go
package main

import (
    "fmt"
)

func main() {
    var nombre string
    var edad int
    fmt.Print("Nombre: ")
    fmt.Scanln(&nombre)
    fmt.Print("Edad: ")
    fmt.Scanln(&edad)
    if edad >= 18 {
        fmt.Println("Es mayor de edad")
    } else {
        fmt.Println("Es menor de edad")
    }
}

File: gpt/nivel intermedio/c++/edad.cpp
#include <iostream>
#include <sstream>
using namespace std;

int main() {
    string nombre;
    int edad;
    cout << "Nombre: ";
    getline(cin, nombre);
    cout << "Edad: ";
    cin >> edad;
    cout << (edad >= 18 ? "Es mayor de edad" : "Es menor de edad") << endl;
    return 0;
}